plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/properties/ObjectPropertyDescriptor.java

```diff
@@ -222,10 +220,21 @@ public boolean isEditable(Object object)
         return getEditableValue(object);
     }
 
+    private List<String> parseFeatures() {
+        String features = propInfo.features();
+        if (features == null || features.isBlank()) {
```

Comment:
Please look at the CommonUtils#isNotEmpty method in the future

Reply:
Or maybe we should make it String[] features() default {};

Reply:
done